### PR TITLE
[FIX] move country=FR from demo data to test file

### PR DESCRIPTION
**Pull request by damdam-s**
see https://github.com/OCA/connector-telephony/issues/79 for more information

### CODE REVIEW

**Comment by damdam-s** [comment]
wait for #80 for lint to be green

**Comment by guewen** [comment]
:+1: 

The error is unrelated. Seems there is a dependency on a non-OCA addon...

**Comment by damdam-s** [comment]
@alexis-via same problem in #80

File: base_phone/__openerp__.py [hunk 1]
```diff
@@ -76,7 +76,7 @@
         'web_phone.xml',
         ],
     'qweb': ['static/src/xml/*.xml'],
-    'demo': ['base_phone_demo.xml'],
+    'demo': [],
```
Comment (Garamotte) [inline base_phone/__openerp__.py]: I think you should remove the empty key.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
@damdam-s can you explain what's happening in the commit message?

**Comment by damdam-s** [comment]
@sylvain-garancher @pedrobaeza : demo key removed
@gurneyalex : case described in commit message

Thanks

**Comment by Garamotte** [comment]
:+1:

**Comment by alexis-via** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Merging for now, and letting the missing module question in the other PR.

**Pull request merged**